tools/cbmc/proofs/TaskPool/TaskCreate/README.md

```diff
@@ -0,0 +1,20 @@
+This proof demonstrates the memory safety of the TaskCreate function.  We
+assume task lists to be initialized, and nondet. set `pxCurrentTCB`,
```

Comment:
Does this mean "and non-deterministically set..."?

Reply:
I'm updating this to read:

```
This proof demonstrates the memory safety of the TaskCreate function.
We initialize task lists, but we set other data structures to
unconstrained (arbitrary) values, including the data structures
`pxCurrentTCB`, `uxCurrentNumberOfTasks`, `pcName` and `pxCreateTask`.
STACK_DEPTH is set to a fixed number (10) since it is not possible to
specify a range.
```